notebook/services/contents/manager.py

```diff
@@ -443,7 +443,7 @@ def check_and_sign(self, nb, path=''):
         if self.notary.check_cells(nb):
             self.notary.sign(nb)
         else:
-            self.log.warning("Saving untrusted notebook %s", path)
+            self.log.warning("Signing notebook %s", path)
```

Comment:
This message is backwards. In this event, the notebook is *not* signed. It is still untrusted.